app/mailers/mail_moderation_mailer.rb

```diff
@@ -1,16 +1,17 @@
 require 'ostruct'
 
 class MailModerationMailer < ApplicationMailer
-  def request_for_moderation_email(user, stored_mail)
-    @user = user
+  def request_for_moderation_email(moderator, stored_mail)
+    @user = moderator
     @sender = stored_mail.sender
     @mail_alias = stored_mail.mail_alias
     @moderator = stored_mail.mail_alias.moderator_group.name
     @subject = stored_mail.subject
     @received_at = stored_mail.received_at
-    @moderation_url = to_moderation_url(stored_mail)
+    @header_text = 'Herinnering'
```

Comment:
This isn't a reminder right?

Reply:
Nice catch, will change to "Moderatieverzoek", shall I change the header text of the reminder mails to "Herinnering moderatieverzoek" for consistency maybe?